lib/travis.js

```diff
@@ -6,6 +6,8 @@ var TRAVIS_BRANCH_CONFIGS = {
     SHOW_DEV_RIBBON: 'on',
     MAILINGLIST_URL: 'https://sendto.mozilla.org/page/s/maker-party-signup-for-teach-site-staging-testing',
     MAILINGLIST_PRIVACY_NAME: 'custom-3508'
+    // PLEDGE_MAILINGLIST_URL: '',
+    // PLEDGE_MAILINGLIST_PRIVACY_NAME: ''
```

Comment:
These serve as a reminder for me.  I'll plug in the corresponding values and uncomment these lines once I know which BSD form to use.